app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -84,7 +87,22 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
                 shareIntent.putExtra(Intent.EXTRA_TEXT, shareSub)
                 startActivity(Intent.createChooser(shareIntent, getString(R.string.msg_share_using)))
             }
+            resources.getString(R.string.title_rate_us) ->
+                startAppPlayStore()
+        }
+    }
 
+    fun getMarketAppLink(): String {
+        return getString(R.string.market_link)
+    }
+    fun getMarketWebLink(): String {
+        return getString(R.string.play_store_link)
+    }
+    private fun startAppPlayStore() {
+        try {
+            startActivity(Intent(Intent.ACTION_VIEW, getMarketAppLink().toUri()))
```

Comment:
It should be better if you move the `getMarketAppLink()` and `getMarketWebLink()` under `startAppPlayStore()` function. Only for a better visual coding.